aws-codeguruprofiler-profilinggroup/sample-template.json

```diff
@@ -5,7 +5,10 @@
     "MyProfilingGroup": {
       "Type": "AWS::CodeGuruProfiler::ProfilingGroup",
       "Properties": {
-        "ProfilingGroupName": "MySampleProfilingGroup"
+        "ProfilingGroupName": "MySampleProfilingGroup",
+        "ChannelUris": [
+          "arn:aws:sns:us-east-1:977364881114:MyTopic"
```

Comment:
There shouldn't be hardcoded ARNs in the template since it's going to fail for anyone deploying this CloudFormation template that does not have permissions to the account `977364881114`.